stile/hsc/base_tasks.py

```diff
@@ -63,7 +63,8 @@ class CCDSingleEpochStileConfig(lsst.pex.config.Config):
                              "WhiskerPlotStar", "WhiskerPlotPSF", "WhiskerPlotResidual",
                              "ScatterPlotStarVsPSFG1", "ScatterPlotStarVsPSFG2",
                              "ScatterPlotStarVsPSFSigma", "ScatterPlotResidualVsPSFG1",
-                             "ScatterPlotResidualVsPSFG2", "ScatterPlotResidualVsPSFSigma"
+                             "ScatterPlotResidualVsPSFG2", "ScatterPlotResidualVsPSFSigma",
+                             "RMSE1Sky", "RMSE2Sky", "RMSE1Chip", "RMSE2Chip", "CountPerMagnitude"
```

Comment:
I'm not sure we need to keep these as separate per-component RMS's.  Normally when I am doing this kind of calculation I'll combine the components, e.g., for a per-component RMS I would do ```0.5*(sum(e1^2)+sum(e2^2))```.

Reply:
Apparently, at some point, I added the `RMSESky` and `RMSEChip` options--thanks for the suggestion.  Should we leave the e1, e2 versions as well (in case we need them as a diagnostic) or just remove them as dead weight in the code?